specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json

```diff
@@ -9497,6 +9497,9 @@
           "$ref": "#/definitions/BlobTags"
         },
         "ObjectReplicationMetadata": {
+          "xml": {
+            "name": "OrMetadata"
+          },
```

Comment:
btw. should this go inside `#/definitions/ObjectReplicationMetadata` ? so that if we reference this elsewhere it won't get lost ?

Reply:
@kasobol-msft It actually is already specified in the ObjectReplicationMetadata definition. Whats weird is that the .NET generator isnt picking it up for some odd reason.

Reply:
@amnguye if so then we should rather be fixing .NET generator. for example:
- (short term) add entry to swagger transforms in .NET sdk (readme.md)
- (long term) fix the generator to so that it sees that from referenced type schema.